### ISSUE TITLE
Incorrect outsideWidth, outsideHeight after Windows wake up?

### ISSUE THREAD

**Issue by geniot** [hajimehoshi/ebiten#2920]
### Ebitengine Version

3eaa03e1

### Operating System

- [X] Windows
- [ ] macOS
- [ ] Linux
- [ ] FreeBSD
- [ ] OpenBSD
- [ ] Android
- [ ] iOS
- [ ] Nintendo Switch
- [ ] PlayStation 5
- [ ] Xbox
- [ ] Web Browsers

### Go Version (`go version`)

1.22rc2

### What steps will reproduce the problem?

1. Run an executable with an Ebiten app
2. Suspend (hibernate) Windows.
3. Wake up.
The working area of the window now has white padding at the bottom and on the right side:
![sizing](https://github.com/hajimehoshi/ebiten/assets/199748/30fa9ccd-450e-445c-b259-18df65642465)

### What is the expected result?

No padding.

### What happens instead?

The working area of the window now has white padding at the bottom and on the right side.

### Anything else you feel useful to add?

If I resize the window (if the window is resizable) the padding disappears.

**Comment by hajimehoshi** [comment]
Does this happen with Ebitengine v2.6.6?

**Comment by geniot** [comment]
You are right! It doesn't. Sorry, the ebiten-ui project uses v2.5.0 and I didn't pay enough attention to that.

**Comment by geniot** [comment]
> Well, but didn't you confirm this happened with [3eaa03e](https://github.com/hajimehoshi/ebiten/commit/3eaa03e19313bf0bf1af2cd5cb44d1cec6801a6f)...?

I made a mistake... I thought that my app which uses the latest Ebiten has the same problem but my padding was actually intentional. Then I didn't test thoroughly. I will be more careful next time. Thank you very much for this wonderful framework. I started to create a desktop learning application funcionally based on this one: https://play.octavian.app/piano And it looks very promising so far.
